### PR TITLE
[charter]proposed initial content

### PR DESCRIPTION
**Pull request by wcarr**
proposed starting point for WebVR specific charter content.

### CODE REVIEW

**Comment by anssiko** [comment]
@wcarr Thanks, I submitted a PR against your repo with some editorial changes, see https://github.com/wcarr/webvr/pull/1

File: charter/index.html [hunk 1]
```diff
@@ -28,269 +28,278 @@
...
-    </p>
-  
-      <h3 id="out-of-scope">
-        Out of Scope
-      </h3>
-    
-      <p class="remove">{TBD: Identify topics known in advance to be out of scope}</p>
-
+    <p>
+      One initial specification the group will work on is the WebVR API that
```
Comment (cvan) [inline charter/index.html]: do we want to link to [the current spec](https://mozvr.github.io/webvr-spec/) or to the [`w3c/webvr` repo](https://github.com/w3c/webvr)?

File: charter/index.html [hunk 2]
```diff
@@ -28,269 +28,278 @@
...
+      defines an API for accessing Virtual Reality (VR) devices, including
+      sensors and head-mounted displays on the Web. Experimental
+      implementations are available for Firefox and Chrome browsers.
```
Comment (cvan) [inline charter/index.html]: to be specific, it's Firefox Nightly release channel and experimental builds of Chromium. not sure the distinction matters much though.

File: charter/index.html [hunk 3]
```diff
@@ -28,269 +28,278 @@
...
       Goals
     </h2>
-
-    <p><span class="remove">{TBD: describe the mission and goals of the Community Group.
-    This should be a brief description describing the reason the group has been formed.}
-    </span></p>
-
+    <p>
+      The Goal of the W3C WebVR Community Group is to help bring
+      high-performance Virtual Reality to the open Web. Currently, access to VR
```
Comment (cvan) [inline charter/index.html]: should probably add " (VR)" for the first time mentioning

File: charter/index.html [hunk 4]
```diff
@@ -28,269 +28,278 @@
...
-  
-      <h3 id="out-of-scope">
-        Out of Scope
-      </h3>
-    
-      <p class="remove">{TBD: Identify topics known in advance to be out of scope}</p>
-
+    <p>
+      One initial specification the group will work on is the WebVR API that
+      defines an API for accessing Virtual Reality (VR) devices, including
```
Comment (cvan) [inline charter/index.html]: and then [if the "Virtual Reality" initialism is displayed above](https://github.com/w3c/webvr/pull/1/files#diff-a69c1e830c6fd14bae30cdc6eef6bc9cR73), feel free to just say "VR" here

File: charter/index.html [hunk 5]
```diff
@@ -28,269 +28,278 @@
...
+      experiences.
+    </p>
+    <p>
+      The new generation of VR headsets benefit from increased external
+      environmental awareness. Depth cameras when integrated into a VR headset
+      or a mobile device enable environment awareness that improves the VR
+      experience significantly. A new generation of VR headsets are emerging
+      that integrate depth sensing into their products (e.g. see the Intel
+      product announced at CES 2016). Another spec in the CG will produce an
+      experimental Web Hand API to allow users to see their hands in the
```
Comment (cvan) [inline charter/index.html]: is this like Leap Motion? couldn't easily find a link to this

File: charter/index.html [hunk 6]
```diff
@@ -28,269 +28,278 @@
...
+      high-quality VR. The WebVR API provides purpose-built interfaces to VR
+      hardware to allow developers to build compelling, comfortable VR
+      experiences.
+    </p>
+    <p>
+      The new generation of VR headsets benefit from increased external
+      environmental awareness. Depth cameras when integrated into a VR headset
+      or a mobile device enable environment awareness that improves the VR
+      experience significantly. A new generation of VR headsets are emerging
+      that integrate depth sensing into their products (e.g. see the Intel
```
Comment (cvan) [inline charter/index.html]: comma after `e.g.`

File: charter/index.html [hunk 7]
```diff
@@ -28,269 +28,278 @@
...
+      product announced at CES 2016). Another spec in the CG will produce an
+      experimental Web Hand API to allow users to see their hands in the
+      virtual environment.
+    </p>
+    <p>
+      The APIs use the web’s security model based on origins, and as such
```
Comment (cvan) [inline charter/index.html]: `the target`

File: charter/index.html [hunk 8]
```diff
@@ -28,269 +28,278 @@
...
-       test suite contribution policy</a> and 
-       <a href="http://www.w3.org/Consortium/Legal/2008/04-testsuite-copyright.html">licenses
-       for W3C test suites></a>. If there are no plans to create a test suite 
-       or other software, please state that.}</p> 
-    
+    <h3 id="specifications">
+      Specifications
+    </h3>
+    <p>
+      <a href="http://mozvr.github.io/webvr-spec/">WebVR API</a>: API for
```
Comment (cvan) [inline charter/index.html]: ah, link here. feel free to use `https:` too

File: charter/index.html [hunk 9]
```diff
@@ -28,269 +28,278 @@
...
+      accessing Virtual Reality (VR) devices, starting from Web pages Incubator
```
Comment (cvan) [inline charter/index.html]: "Web pages Incubator" link?

Comment (cvan) [inline charter/index.html]: just "VR" should be fine

File: charter/index.html [hunk 10]
```diff
@@ -28,269 +28,278 @@
...
       Dependencies or Liaisons
     </h2>
-    
-    <p class="remove">{TBD: List any significant dependencies on other groups 
-      (inside or outside W3C) or materials. }
-    </p>    
-    
+    <p class="remove">
+      {TBD: List any significant dependencies on other groups (inside or
+      outside W3C) or materials. }
```
Comment (cvan) [inline charter/index.html]: remove space before `}`

File: charter/index.html [hunk 11]
```diff
@@ -28,269 +28,278 @@
...
     </h1>
-
-    <p><span class="remove">{TBD: remove next sentence before submitting for approval} </span>
-    This Charter is work in progress. To submit feedback, please use 
-    <span class="remove">{TBD:GitHub repository URL}</span> Issues where Charter is being developed.</p>
-    
+    <p>
+      <span class="remove">{TBD: remove next sentence before submitting for
+      approval}</span> This Charter is work in progress. To submit feedback,
+      please use <span class="remove">{TBD:GitHub repository URL}</span> Issues
```
Comment (cvan) [inline charter/index.html]: that should be https://w3c.github.io/webvr/charter/ (or https://github.com/w3c/charter/issues), right?

**Comment by cvan** [comment]
this looks quite good! thanks for taking the time to add all these text. this process is new to most of us. much appreciated :+1:

I'm comfortable merging after those nits are addressed, and we can fill in the gaps. thoughts?

File: charter/index.html (hunk 1 above)
Comment (wcarr) [inline charter/index.html]: @cvan ,  eventually should be on [w3c/webvr](https://github.com/w3c/webvr), but the spec hasn't been copied there yet.

**Comment by wcarr** [comment]
I made the changes initially in the wrong branch, not the patch :)  I've repeated them in patch-1

File: charter/index.html (hunk 7 above)
Comment (wcarr) [inline charter/index.html]: I didn't know what you meant by the target so didn't change the text yet for this one

**Comment by wcarr** [comment]
@cvan I think all the comments are addressed except the one that says "the target". [https://github.com/w3c/webvr/pull/1#discussion_r57794202](https://github.com/w3c/webvr/pull/1#discussion_r57794202)
I didn't know what change that one was asking for.  I suggest that you commit the other changes and we take dare of that remaining one in the next round of changes.

File: charter/index.html (hunk 9 above)
Comment (anssiko) [inline charter/index.html]: I think this should say `API for accessing VR devices from Web pages (Incubator Draft spec).`

The "Incubator Draft spec" referring to this spec to be used as input for incubation in this CG. I'll land a fix to this separately.

**Comment by anssiko** [comment]
@cvan Thanks for the review. I believe all your comments have been addressed by @wcarr in 5e50411 and e49e200. I'll merge this PR now so we can iterate on it further. You should also have privileges to merge PRs -- please let me know if you run into issues with that.

**Pull request merged**